.pre-commit-config.yaml

```diff
@@ -72,7 +72,14 @@ repos:
     rev: v0.950
     hooks:
     -   id: mypy
-        pass_filenames: false
+        name: mypy-habitat-lab
+        files: habitat-lab
```

Comment:
This selection can and should by editing the config for mypy?

Reply:
This is using the pre-commit mypy, not mypy. I would like to have these two separate in pre-commit as we are trying to separate habitat-lab and habitat-baselines. Eventually, I would want the test suites and CI to be separate as well.